projekt-03-kontakty-main/projekt-03-kontakty-main/style.css

```diff
@@ -8,3 +8,44 @@ body {
 }
 
 /* ---------------------------------------------------- */
+
+.container {
+	display: inline-flexbox;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-content: center;
+	background-color: whitesmoke;
+	border-radius: 1rem;
+	align-items: stretch;
+}
+
+.item {
+	display: inline-block;

```

Comment:
Tady by `inline-block` nebyl potřeba, aby se ti kontakty řadily vedle sebe. To už umí zajistit sám flexbox, ale Tobě to nefungovalo, protože jsi neměla v .container validní hodnotu pro display

Reply:
Když už pracujeme s flexboxem, který nám trošku usnadňuje práci, nebála bych se nastavit `display: flex;` i tu, ať se nám item chová jako flexbox i vůči svému obsahu (fotka, jméno, popis...)
Může to pak ušetřit práci díky možnostem, které flexbox nabízí 🙂